src/bundle/Resources/views/themes/admin/content/edit/base.html.twig

```diff
@@ -45,7 +45,7 @@
                                             {% if formField.value is defined %}
                                                 {{- form_widget(formField) -}}
                                             {% else %}
-                                                <div>
+                                                <div class="ez-content-edit--ezcontent-query">
```

Comment:
This is not a field type (SCSS for this are in the field type group)
Probably this class is not proper one. 
Why this is with modifier? Is it a `ez-content-edit` container?